data-hibernate-jpa/src/test/java/io/micronaut/data/hibernate/entities/EventIndividualTest.java

```diff
@@ -25,6 +26,7 @@ public class EventIndividualTest {
     @DateUpdated
     private LocalDateTime dateUpdated;
 
+    @Column(name = "int_value")
```

Comment:
`value` is reserved keyword and column cannot be named `value` in new h2 version.[This discussion](https://stackoverflow.com/questions/70767876/h2hibernate-error-when-the-column-name-is-a-keyword) says that we can add property to JDBC url to override that behavior but I couldn't figure out where is JDBC url being constructed for hibernate jpa tests.